internal/metrics/remote_write.go

```diff
@@ -153,19 +160,58 @@ func (r *RemoteWrite) Update(config models.DeviceConfigurationMessage) error {
 			return fmt.Errorf("metrics remote write configuration is invalid. RequestNumSamples has to greater than 0")
 		}
 
+		// create TLS client config
+		clientConfig := config_util.HTTPClientConfig{}
+
+		if newConfig.CaCert != "" {
+			// create new file for CA. can't use same file cause client watches for changes in the file we pass to it.
+			caFile, err := ioutil.TempFile(r.dataDir, ServerCAFileNamePattern)
+			if err != nil {
+				log.Error("could not create temp file for metics remote write server CA")
```

Comment:
Should we log the failing filepath?

Reply:
the function, TempFile, creates the path. So when it fails I do not have a path to log. However, I can add the dataDir. Also, I just noticed that I have a spelling error. metics ==> metrics. I don't know why my spell checker misses so much.